index.html

```diff
@@ -93,6 +96,51 @@ <h1 class="font-family-proza inline-block new">Entries</h1>
         </div>
       </div>
     </div>
+    <div data-view="edit-entry-form" class="view">
+      <div class="container">
+        <div class="row">
+          <div class="column-full">
+            <h1 class="font-family-proza">Edit Entry</h1>
+          </div>
+        </div>
+        <form id="edit-form-journal">
+          <div class="row">
+            <div class="column-half">
+              <img src="images/placeholder-image-square.jpg" id="editImageDisplay" alt="Journal Image">
+            </div>
+            <div class="column-half">
+              <div>
+                <label class="font-family-proza" for="editTitle">Title</label>
+              </div>
+              <div>
+                <input type="text" id="editTitle" name="editTitle" required>
+              </div>
+              <div>
+                <label class="font-family-proza" for="editPhotoUrl">Photo URL</label>
+              </div>
+              <div>
+                <input type="text" id="editPhotoUrl" name="editPhotoUrl" required>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-full">
+              <div>
+                <label class="font-family-proza" for="editNotes">Notes</label>
+              </div>
+              <div>
+                <textarea name="notes" id="editNotes" rows="8" required></textarea>
+              </div>
+            </div>
+          </div>
+          <div class="row">
+            <div class="column-full">
+              <button type="submit" class="font-family-proza">SAVE</button>
+            </div>
+          </div>
+        </form>
+      </div>
+    </div>
```

Comment:
You should not be making an additional view for editing, you should just be updating the values in the `entry-form` view with the current values for the entry being edited.